### DISCUSSION TITLE
An `anomalib` Starter for Newcomers in Kaggle environment.

### DISCUSSION THREAD

**Discussion by innat** [open-edge-platform/anomalib#266] [Show and tell]
# 📢  Starter Notebook

We've made a starter notebook for newcomers about how to use `anomalib` library in a kaggle environment. It's a minimal starter notebook, hope newcomers find it useful. 

- [MVTec-AD: Anomaly Detection with Anomalib Library](https://www.kaggle.com/code/ipythonx/mvtec-ad-anomaly-detection-with-anomalib-library)

![image](https://user-images.githubusercontent.com/17668390/164736543-0ef58c66-dfb0-47c5-8e6e-26774dbc6fd3.gif)

**Comment by innat** [comment]
cc. @samet-akcay

**Comment by samet-akcay** [comment]
@innat this is awesome! Thanks a lot for your contribution!
